### PR TITLE
fix(lora): sync PEFT scaling state to EMA

### PR DESCRIPTION
**Pull request by onion-hong**
## Summary

- sync PEFT LoRA `scaling` dictionaries from the online model to EMA
- preserve the existing fallback LoRA EMA synchronization behavior
- cover validation and checkpoint serialization with regression tests
- document Brain Tumor and VisDrone rank-sweep results collected while reproducing the issue

## Problem

With PEFT 0.19.1 and `lora_alpha_warmup` enabled, the online LoRA model updates its scaling values during training, but the EMA model keeps the initial zero scaling.

PEFT stores this state in a regular Python dictionary rather than a tensor registered in `state_dict`, so the standard EMA update does not copy it.

This causes training loss to decrease while EMA validation metrics collapse, and saved EMA checkpoints may effectively evaluate with LoRA disabled.

Observed state before the fix:

```text
online scaling: {"default": 3.0}
EMA scaling:    {"default": 0.0}

### CODE REVIEW

**Comment by isLinXu** [comment]
LGTM
